stdlib/public/core/Set.swift

```diff
@@ -712,7 +712,8 @@ extension Set: SetAlgebra {
   @inlinable
   public func isSubset<S: Sequence>(of possibleSuperset: S) -> Bool
   where S.Element == Element {
-    // FIXME(performance): isEmpty fast path, here and elsewhere.
```

Comment:
As noted in the comment, there are places "elsewhere" that could use an `isEmpty` fast path. Before deleting the fixme, it'd be important to identify where those other places are and either implement the fast path or add the fixme comment there.

Reply:
For sure @xwu 👍 
I've checked the methods on Set and added it in all places that I could find it would be possible :))

Reply:
I've check this `isSuperset`, but didn't think it has benefit, is an interesting case because just because we have two cases there
```swift
let s: Set<String> = []

print(s.isSuperset(of: [])) //false
print(s.isSuperset(of: ["A", "B"])) //true
```
So to add an `isEmpty` fast path will require, check also possibleSubset for empty too, but since it is a arbitrary sequence it doens't have an `isEmpty` and as far as I know we could not do this check on underestimatedCount.

Also looking to the implementation(below) no matter what is the size of `possibleSubset` if self is empty it will return on the first iteration because contains will return false. So there's no performance problem because it will not iterate unnecessarily. Basically, that is the reason I think is not a benefit :)) 
```swift
public func isSuperset<S: Sequence>(of possibleSubset: __owned S) -> Bool
    where S.Element == Element {
    for member in possibleSubset {
      if !contains(member) {
        return false
      }
    }
    return true
  }
```

Reply:
Turns out there _was_ more "elsewhere": `isStrictSubset` and `isStrictSuperset` with `Sequence` parameter could probably also benefit form optimizations for `isEmpty` cases... see https://github.com/apple/swift/pull/24156#issuecomment-489604680.

Reply:
@palimondo Sorry I miss those 😅, I'll take a look and I can patch up a PR later today :))

Reply:
No problem. Please don't go in there now, #21300 is about to drastically change how that's handled and I'm already looking into how to handle `isEmpty` cases on top of it…

I just couldn't resist noting how @xwu's pedantry usually turns out to be right, even though it feels pretty annoying to be on the receiving end of it (speaking from experience 😉).